YouTube/scripts/addVideoElement.js

```diff
@@ -0,0 +1,32 @@
+const template = document.querySelector("#list-item");
+
+function addVideoElements(totalWidth, videoArray, itemsPerPage) {
+
+  const fragment = document.createDocumentFragment();
+  videoArray.forEach((videoItem) => {
+    const { thumbnail, author, title, description, publishTime, views, videoId } = videoItem;
```

Comment:
Please extract to a separate function.

Reply:
extracted in separate function createVideoElement